src/Api/Groups.php

```diff
@@ -205,6 +205,35 @@ public function removeMember($group_id, int $user_id)
     {
         return $this->delete('groups/'.self::encodePath($group_id).'/members/'.self::encodePath($user_id));
     }
+    
+    /**
+     * @param int|string $group_id
+     * @param array      $parameters
```

Comment:
Please add phpdoc for this. I cannot accept this PR without this. Please see the other methods for an example.

Reply:
Done